osudroidresource/src/main/res/values-pt-rBR/options.xml

```diff
@@ -6,59 +6,223 @@
+    <string name="opt_directory_summary">Caminho do diretório de beatmaps (padrão: /mnt/sdcard/osu!droid/Songs)</string>
+
+    <string name="opt_clear_title">Cache da biblioteca</string>
+    <string name="opt_clear_summary">Força a limpeza do cache da biblióteca</string>
+
+    <string name="opt_doublesound_title">Hitsounds melhores</string>
+    <string name="opt_doublesound_summary">Faz com que os sons fiquem mais semelhantes ao do osu! original</string>
+
     <string name="opt_sound">Som</string>
-    <string name="opt_fps_title">Mostrar FPS</string>
-    <string name="opt_fps_summary">Mostra taxa de quadros por segundos (FPS) durante o jogo.</string>
-	<string name="opt_lowtextures_title">Reduzir qualidade das texturas.</string>
-    <string name="opt_lowtextures_summary">Reduz a qualidade das texturas do jogo. Se desativado, o jogo passara a utilizar as texturas de alta resolução (2x).</string>
-    <string name="opt_nativeplayer_title">Utilizar o reprodutor integrado</string>
-    <string name="opt_nativeplayer_summary">Utilizar o reprodutor integrado para maior precisão (arquivos .ogg não suportados).</string>
-    <string name="opt_bgbrightness_title">Brilho do fundo</string>
-    <string name="opt_bgbrightness_summary">Altera o brilho de fundo</string>
+
+    <string name="opt_fps_title">Mostrar o FPS</string>
+    <string name="opt_fps_summary">Mostra o FPS (Frames Por Segundo) durante a gameplay</string>
+
+    <string name="opt_skipoffset_title">Offset de skip</string>
+    <string name="opt_skipoffset_summary">Esse offset vai ser adicionado quando você pressionar o botão de &quot;Skip&quot;. Coloque -1 para um offset automático</string>
+
+    <string name="opt_lowtextures_title">Qualidade da textura</string>
+    <string name="opt_lowtextures_summary">Reduz a qualidade das texturas. Se desativado, O osu!droid ira utilizar Texturas em resolução HD.</string>
+
+    <string name="opt_nativeplayer_title">Usar o player embutido</string>
+    <string name="opt_nativeplayer_summary">Utiliza do player embutido para um timing mais preciso (não suporta arquivos .ogg)</string>
+
+    <string name="opt_metronome_switch_title">Switch de metrônomo</string>
+    <string name="opt_metronome_switch_summary">Ativa o efeito sonoro de metrônomo (Apenas no mod nightcore)</string>
+
     <string name="opt_vbroffset_title">Offset de VBR</string>
-    <string name="opt_vbroffset_summary">Offset para arquivos mp3 codificados com uma taxa de bits (bitrate) variável (sólo para reproductor integrado)</string>
-    <string name="opt_sliderborders_title">Mostrar borda dos sliders</string>
-    <string name="opt_sliderborders_summary">Mostra as bordas dos sliders. Desativar para maior desempenho.</string>
+    <string name="opt_vbroffset_summary">Offset para arquivos mp3 codificados com um bitrate váriavel (Apenas com o player embutido)</string>
+
+    <string name="opt_sliderborders_title">Mostrar bordas nos slider</string>
```

Comment:
Shouldn't this just be ``Bordas em sliders`` (or something similar)?

Reply:
done

Reply:
and yes it should be